### DISCUSSION TITLE
Is Onboarding PR detectable in a template

### DISCUSSION THREAD

**Discussion by astellingwerf** [renovatebot/renovate#11681] [Request Help]
I want to able to (in webhook) distinguish the Renovate PRs, so I configured the following:
```
  "addLabels": [
    "renovate-update-{{updateType}}"
  ],
```

This works great for many of the PRs, but not for the Onboarding. The `updateType` field will not contain the value 'onboarding', and there is no `isOnboarding` field either. Either can this be added, or is there another im- or explicit way to detect the Onboarding PR?

**Comment by rarkins** [comment]
It has a fixed PR title ("Configure Renovate")

**Reply by astellingwerf** [reply]
(Or whatever is configured in `onboardingPrTitle`..) But it will also have a label `renovate-update-{{updateType}}`? There's nothing I can do about that?

**Reply by rarkins** [reply]
An onboarding PR does not have an `updateType` so if there's one there then it's a mistake
